src/components/assets/Account.vue

```diff
@@ -69,7 +72,7 @@
         <span>{{ $t('assets.totalBalance') }}</span>
         <q-skeleton v-if="isSkeleton" animation="fade" class="skeleton--md" />
         <span v-else class="text--total-balance">
-          ${{ $n(balUsd + ttlErc20Amount + ttlNativeXcmUsdAmount) }}
+          ${{ $n(balUsd || 0 + ttlErc20Amount + ttlNativeXcmUsdAmount) }}
```

Comment:
It seems like the total amount hasn't included ERC20 tokens.
![image](https://user-images.githubusercontent.com/92044428/174631634-0732f1e1-f9a7-475d-8784-8b9470bbf5fe.png)

Reply:
This was a fix for a TS error that was happening on this file. I'll revert

Reply:
Sure, but I'm not sure the error comes from this line. Please check the total USD amount with an account that has some ERC20 tokens.